realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyMediatorGenerator.kt

```diff
@@ -37,7 +37,7 @@ class RealmProxyMediatorGenerator(private val processingEnvironment: ProcessingE
     private val primaryKeyClasses = mutableListOf<QualifiedClassName>()
 
     init {
-        for (metadata in classesToValidate) {
+        for (metadata in classesToValidate.toSortedSet(compareByDescending { it.qualifiedClassName.name })) {
```

Comment:
Out of curiousity, what type of `Set` is the set that is passed in here?

Reply:
A `SortedSet`:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/to-sorted-set.html

Reply:
I meant, what is the type of `classesToValidate` which is passed to the constructor of `RealmProxyMediatorGenerator`? I assume it is not already sorted.

Reply:
makes sense :)
This is just a Set

Reply:
Thanks, discussed this offline. This looks good!